src/pickley/bstrap.py

```diff
@@ -14,32 +14,40 @@
 from urllib.error import HTTPError, URLError
 from urllib.request import Request, urlopen
 
+DEFAULT_BASE = "~/.local/bin"
 DOT_META = ".pk"
 DRYRUN = False
 HOME = os.path.expanduser("~")
 PICKLEY = "pickley"
+PIP_CONFS = ("~/.config/pip/pip.conf", "/etc/pip.conf")
```

Comment:
To do this properly is actually this whole mess, do you document that only a subset work for bootstrap?

https://github.com/python-packaging/indexurl/blob/cc77b2d746f03fc05d066882478cc1734b0c15ef/indexurl/core.py#L56-L72

Reply:
Yeah, I'm doing the 80% here :)
I should document that pickley will try and respect an existing `pip.conf`, but only from those 2 locations